website_sale_secondary_unit/tests/test_website_sale_secondary_unit.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests.common import HttpCase
+
+
+class WebsiteSaleHttpCase(HttpCase):
+    def setUp(self):
+        super(WebsiteSaleHttpCase, self).setUp()
+        with self.cursor() as cr:
+            env = self.env(cr)
```

Comment:
Do not use this `with` trick anymore. In v12 it's not needed and in v10-11 it's broken now.